lib/railway_ipc/message_consumption.ex

```diff
@@ -49,6 +49,17 @@ defmodule RailwayIpc.MessageConsumption do
 
   def persist_message({:ok, message_consumption}) do
     case ConsumedMessageContext.create(message_consumption) do
+      :no_uuid ->
+        %{inbound_message: %{decoded_message: decoded_message}} = message_consumption
+        metadata = [
+          feature: "ipc_republish_message",
```

Comment:
We want this out of here right? Seems strange and not accurate of the state in the meta at this point of time.

Reply:
I removed this code